doc/dev/design/FABRID.rst

```diff
@@ -0,0 +1,674 @@
+********
+FABRID
+********
+.. _fabrid-design:
+
+- **Author**: Justin Rohrer, Jelte van Bommel, Marc Odermatt, Marc Wyss, Cyrill Krähenbühl, Juan A. García-Pardo
+- **Last updated**: 2024-05-03
+- **Discussion at**: -
+
+Abstract
+===========
+
+In SCION, endhosts have the option to choose *inter-AS* paths to forward packets to their desired destination.
+However, some applications require more fine-grained path selection like "Do not route traffic over devices
+produced by hardware manufacturer X or that run software Y with version < Z" which requires transparency and
+control also with respect to *intra-AS* paths.
+Such fine-grained path selection is useful for example if there exists a known bug in certain router versions that affect secure communication,
+or if an entity simply does not trust a certain hardware manufacturer, so that traffic can be steered around those devices.
+This can also be seen as an enhancement for keeping traffic within a certain jurisdiction, e.g., by routing traffic
+only along devices located in a specific country, because a single AS could cover multiple countries.
+
```

Comment:
- Move the figure "Example network topology" to the end of the abstract

Reply:
Done